oozie-to-airflow/mappers/prepare_mixin.py

```diff
@@ -55,10 +56,7 @@ def parse_prepare_node(oozie_node: ET.Element, params: Dict[str, str]) -> Tuple[
             # If there exists a prepare node, there will only be one, according
             # to oozie xml schema
             for node in prepare_nodes[0]:
-                node_path = el_utils.replace_el_with_var(node.attrib["path"], params=params, quote=False)
-                if "//" in node_path:
-                    node_path = node_path.split("//", maxsplit=1)[1]  # Removing the hdfs:// or similar part
-                node_path = "/" + node_path.split("/", maxsplit=1)[1]  # Removing the 'localhost:8082/' part
+                node_path = normalize_path(node.attrib["path"], params=params)
```

Comment:
I'm not sure. As far as I know, only paths in URL format are supported by Oozie.

Reply:
It works fine if the URI begins with the scheme, e.g. `http://` or `hdfs://`, however I tested with `nameNode` value of `localhost:8082` and the result in the DAG is:
```
bash_command='$DAGS_FOLDER/../data/prepare.sh -c cluster-o2a -r europe-west3 -d "8082/examples/output-data/demo/pig-node" -m "8082/examples/input-data/demo/pig-node"',
```

Reply:
Maybe we shouldn't handle such a case but print an error when the scheme is not present in the `path` attribute of the prepare node?
We could stop the conversion and allow the user to correct it.
WDYT?

Reply:
I looked at documentation and found a note about it:
> In the move command the source path must exist. The following scenarios are addressed for a move:
>
> The file system URI(e.g. hdfs://{nameNode}) can be skipped in the target path. It is understood to be the same as that of the source. But if the target path does contain the system URI, it cannot be different than that of the source.
https://oozie.apache.org/docs/5.1.0/WorkflowFunctionalSpec.html#a3.2.4_Fs_HDFS_action
So we should throw an exception only in some cases.

I will update a PR.

Reply:
I update a PR.